src/common/obj_bencher.cc

```diff
@@ -443,8 +444,9 @@ int ObjBencher::write_bench(int secondsToRun,
   lock.Lock();
   data.finished = 0;
   data.start_time = ceph_clock_now();
+  stopTime = data.start_time + runtime;
   lock.Unlock();
-  for (int i = 0; i<concurrentios; ++i) {
+  for (int i = 0; i<concurrentios && ceph_clock_now() < stopTime; ++i) {
```

Comment:
`aio_write()` is a non-blocking call. so it should completes in a jiffy. i think we are more interested in the time taken to **complete** the requests instead of the time to send them.

what do you think?